src/components/document/Tags.js

```diff
@@ -242,83 +244,110 @@ function Tags({ toolbarButtons }) {
+                                        />
+                                    </TagTooltip>
+                                ))
+                            }
+                            inputValue={inputValue}
+                            onInputChange={handleInputChange}
+                            renderInput={params => (
+                                <TextField
+                                    {...params}
+                                    placeholder={(tagsValue.length === 0 ? 'no tags, ' : '') + 'start typing to add'}
+                                    InputProps={{
+                                        ...params.InputProps,
+                                        endAdornment: (
+                                            <React.Fragment>
+                                                {tagsAggregationsLoading ? <CircularProgress color="inherit" size={20} /> : null}
+                                                {params.InputProps.endAdornment}
+                                            </React.Fragment>
+                                        ),
+                                        onFocus: handleInputFocus
+                                    }}
                                 />
-                            </TagTooltip>
-                        ))
-                    }
-                    inputValue={inputValue}
-                    onInputChange={handleInputChange}
-                    renderInput={params => (
-                        <TextField
-                            {...params}
-                            placeholder={(tagsValue.length === 0 ? 'no tags, ' : '') + 'start typing to add'}
-                            InputProps={{
-                                ...params.InputProps,
-                                endAdornment: (
-                                    <React.Fragment>
-                                        {tagsAggregationsLoading ? <CircularProgress color="inherit" size={20} /> : null}
-                                        {params.InputProps.endAdornment}
-                                    </React.Fragment>
-                                ),
-                                onFocus: handleInputFocus
-                            }}
+                            )}
+                            onChange={handleChange}
                         />
-                    )}
-                    onChange={handleChange}
-                />
+                    </Grid>
+
+                    <Grid item style={{ marginLeft: 20 }}>
+                        <Tooltip
+                            interactive
+                            classes={{ tooltip: classes.noMaxWidth }}
+                            title={
+                                <>
+                                    <Typography variant="body2">
+                                        Tags are made of lowercase ASCII letters, digits, and symbols:{' '}
```

Comment:
Let's move the actual texts (or JSX elements I guess) into a separate file called "help".

For this section, we should link here: https://github.com/liquidinvestigations/docs/wiki/User-Guide:-Hoover#tags 

We can put it a final line in the tooltip:`See more <a>here</a>.`

Reply:
done